airflow/providers/docker/operators/docker.py

```diff
@@ -187,7 +187,14 @@ def __init__(
     ) -> None:
         super().__init__(**kwargs)
         self.api_version = api_version
-        self.auto_remove = auto_remove
+        if str(auto_remove) == "False":
+            self.auto_remove = "never"
+        elif str(auto_remove) == "True":
+            self.auto_remove = "success"
+        elif str(auto_remove) in ("never", "success", "force"):
+            self.auto_remove = str(auto_remove)
```

Comment:
```suggestion
            self.auto_remove = auto_remove
```

Doesn’t seem to be needed?